### DISCUSSION TITLE
Fresh Install iOS Invalid Installation URL

### DISCUSSION THREAD

**Discussion by datzent83** [chatwoot/chatwoot#6396] [Self hosted]
Just did a fresh install of ChatWoot on Ubuntu server. I configured everything to the letter. The website works just fine, but when I put the domain of my self hosted installation I get "Invalid Installation URL" I put the exact same domain I am using on the web browser. I made sure Nginx is properly configured and setup. Not sure what I am missing.

**Comment by sojan-official** [comment]
@datzent83  could you add a screenshot of where you are getting the error ?

**Comment by vishnu-narayanan** [comment]
Have you set up SSL?

**Reply by datzent83** [reply]
Yes. I have setup SSL
